src/nft/pages/explore/index.tsx

```diff
@@ -0,0 +1,11 @@
+import Banner from '../../components/explore/Banner'
+
+const NftExplore = () => {
+  return (
+    <>
+      <Banner />
+    </>
+  )
+}
```

Comment:
There's going to be more content here soon, right? This isn't just a passthrough wrapper?

Reply:
Right, I'm migrating things over in pieces to have smaller PRs